.circleci/config.yml

```diff
@@ -83,7 +91,73 @@ release: &release
         name: Release artifacts
         command: ./sbt ++${SCALA_VERSION}! clean sonatypeBundleClean +publishSigned sonatypeBundleRelease
 
+mdoc: &mdoc
+  steps:
+    - checkout
+    - <<: *load_cache
+    - run:
+        name: Generate documentation
+        command: |
+          ./sbt doc
+          ./sbt ++${SCALA_VERSION}! docs/mdoc
+    - <<: *save_cache
+
+install_yarn: &install_yarn
+  - run:
+      name: Install Yarn
+      command: |
+        curl -sL https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
+        sudo bash -c 'echo "deb https://dl.yarnpkg.com/debian/ stable main" > /etc/apt/sources.list.d/yarn.list'
+        sudo apt update && sudo apt install yarn -y
+        yarn policies set-version
+        yarn -v
+
+install_nodejs: &install_nodejs
+  - run:
+      name: Install node.js
+      command: |
+        export NVM_DIR="/opt/circleci/.nvm"
+        [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+        nvm install
+        nvm use
+        node -v
+
+microsite: &microsite
+  steps:
+    - add_ssh_keys:
+        fingerprints:
+          - "11:2c:fa:98:69:d8:a1:d5:76:2e:38:d0:91:39:85:ce"
```

Comment:
We need to change this fingerprint

Reply:
which one shoud i use ?

Reply:
I need to create a new one for this repo will get it done and either message you with it or do a PR to this branch/PR for you

created this to track/remind myself: https://github.com/zio/zio-ftp/issues/22

Reply:
@regis-leray done. Can you change it in this PR to: 

```
68:95:b1:fc:a3:eb:4e:85:45:74:07:d3:c0:35:c8:ec
```

Thanks.

Reply:
thanks i did it